vulcanai2/tests/models/test_cnn.py

```diff
@@ -204,4 +204,14 @@ def test_params_multi_input(self, multi_input_cnn,
             for param, opt_param in zip(test_net1.parameters(),
                                         test_net1.optim.param_groups[0]['params']):
                 assert param is opt_param
-        
\ No newline at end of file
+        
+        # Check the params after saving loaading
+        test_net2.save_model()
+        save_path = test_net2.save_path
+        loaded_test_net2 = BaseNetwork.load_model(load_path=save_path)
+        # Warning: Repeated run of this test will resulting dumping
```

Comment:
No you need to delete the saved model instantly. Nothing should persist after running the test. See the tabular dataset csv saving test for an example of saving, loading and deleting.